internal/support/cronscheduler/config/config.go

```diff
@@ -25,6 +26,11 @@ type WritableInfo struct {
 	Telemetry       bootstrapConfig.TelemetryInfo
 }
 
+type Scheduler struct {
+	// AutoTrigger is a flag to indicate whether the scheduler should automatically trigger the missed scheduled jobs when the service starts
+	AutoTrigger bool
```

Comment:
move this config to the job DTO/Model and renamed to `autoTriggerMissedRecords`
the default can be false